inc/policy.class.php

```diff
@@ -190,4 +190,28 @@ public function getSearchOptionsNew() {
       return $tab;
    }
 
+   public static function dropdown($options = []) {
+      global $DB;
+
+      $request = [
+         'FROM' => PluginFlyvemdmPolicyCategory::getTable(),
+      ];
+
+      $category = [];
+      foreach ($DB->request($request) as $row) {
+         $elements[$row['name']] = [];
+         $category[$row['id']] = $row['completename'];
+      }
+
+      $request = [
+         'FROM' => static::getTable(),
+      ];
+      $currentCategory = '';
+      foreach ($DB->request($request) as $row) {
+         $categoryName = $category[$row['plugin_flyvemdm_policycategories_id']];
+         $elements[$categoryName][] = $row['name'];
+      }
+      return Dropdown::showFromArray(static::getForeignKeyField(), $elements, $options);
+   }
+
```

Comment:
redefines the default implementation of dropdown() to show policies sorted by category